data/Molcas/Molcas/C_bigbasis.in

```diff
@@ -0,0 +1,11 @@
+&gateway
```

Comment:
This comment isn't about the contents of the file, but generally about the path for the test data. It should be more specific than `Molcas/Molcas`, because at some point people may want to parse older versions of full Molcas (6, 7, 8 for example), and OpenMolcas may have breaking changes later.

For now, something like `Molcas/basicMolcas8.0` is enough, and then new files with OpenMolcas can be `Molcas/basicOpenMolcas18.0`.

Reply:
Yup, please add the version to the path.